### PR TITLE
Add config helper

### PR DESCRIPTION
**Pull request by ericgaspar**
- Add ynh_add_config helper

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: doc/DISCLAIMER.md [hunk 1]
```diff
@@ -1,72 +1,41 @@
-## Disclaimer
-
-This package installs Discourse without Docker, for several reasons (mostly to support ARM architecture and low-profile servers, to mutualize nginx/postgresql/redis services and to simplify e-mail setup).
-As stated by the Discourse team:
-> The only officially supported installs of Discourse are [Docker](https://www.docker.io/) based. You must have SSH access to a 64-bit Linux server **with Docker support**. We regret that we cannot support any other methods of installation including cpanel, plesk, webmin, etc.
```
Comment (alexAubin) [inline doc/DISCLAIMER.md:5]: Everything LGTM but not sure why removing this disclaimer ? It's a bit long as hell but sounds legit to have this

The french version still has it at the very top

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!tastemetitus

**Comment by ericgaspar** [comment]
!testme

**Comment by weex** [comment]
@JimboJoe Any thoughts as a requested reviewer? I may try installing and testing in the coming weeks.

**Comment by yalh76** [comment]
!testme

**Pull request closed without merging**